src/Business Foundation/App/NoSeries/Permissions/NoSeriesAdmin.permissionset.al

```diff
@@ -0,0 +1,24 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
+permissionset 304 "No. Series - Admin"
+{
+    Access = Internal;
+    Assignable = false;
+    IncludedPermissionSets = "No. Series - View";
+
+    Permissions =
+        tabledata "No. Series" = IMD,
```

Comment:
need to update and verify all the permissionsets

Reply:
I have created a workitem for this.